content/docs/orgs/restricting-users-to-ip-ranges.md

```diff
@@ -0,0 +1,19 @@
+---
+menu:
+  docs:
+    parent: orgs
+title: Restricting users to trusted IP ranges
+---
+
+We can configure cloud.gov to restrict users in a particular domain (e.g., `gsa.gov`) from using cloud.gov services unless their requests originate from a trusted network location. Your users will still be able to connect to cloud.gov from other locations, but any requests they make will be rejected. You can use this feature to help your agency comply with the [Trusted Internet Connections (TIC) Initiative](https://www.dhs.gov/trusted-internet-connections) while using cloud.gov. 
+
+### Contact support to configure restrictions for your domain
+Create a [support ticket](mailto:cloud-gov-support@gsa.gov?body=Email%20domain%3A%0A%0AEgress%20IP%20ranges%3A%0A%0AAgency%20CIO%3A%0A) specifying the IP address ranges that are valid for your domain. Because address restriction applies to all cloud.gov users from your email domain, we will request confirmation from your agency CIO before changing the configuration.
```

Comment:
Still needs work in the template. Customers should be able to cc an agency CIO and have them receive enough context to approve easily.